packages/main/src/plugin/acp/acp-session-manager.ts

```
@@ -0,0 +1,964 @@
+
+    this.apiSender.send('acp-session-update');
+    return info;
+  }
+
+  private async startAcpSession(sessionId: string, prompt: string): Promise<void> {
+    const session = this.sessions.get(sessionId);
+    if (!session) {
+      throw new Error(`Session "${sessionId}" not found`);
+    }
+
+    console.log(`[ACP] initializing connection...`);
```

Biome (2.4.16)

[error] Do not use template literals if interpolation and special-character handling are not needed.

(lint/style/noUnusedTemplateLiteral)